src/sentry/search/eap/resolver.py

```diff
@@ -305,15 +318,83 @@ def _resolve_having(
             return parsed_terms[0], resolved_contexts
         return None, []
 
+    def resolve_virtual_context_term(
+        self,
+        term: str,
+        raw_value: str | list[str],
+        resolved_column: ResolvedColumn,
+        context: VirtualColumnDefinition,
+    ) -> list[str] | str:
+        # Convert the term to the expected values
+        final_raw_value: str | list[str] = []
+        resolved_context = context.constructor(self.params)
+        reversed_context = {v: k for k, v in resolved_context.value_map.items()}
+        if isinstance(raw_value, list):
+            new_value = []
+            for raw_iterable in raw_value:
+                if context.default_value and context.default_value == raw_iterable:
+                    # Avoiding this for now, while this could work with the Unknown:"" mapping
+                    # But that won't work once we use the VirtualColumnContext.default_value
+                    raise InvalidSearchQuery(
+                        f"Using {raw_iterable} in an IN filter is not currently supported"
+                    )
```

Comment:
Why is the default value special when used in an `IN` filter?

Reply:
good catch, i'll disable in both sides for now and re-enable when i fix